### PR TITLE
fix credentials for hitting import service

### PR DESCRIPTION
**Pull request by malaffoon**
add README for task service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,13 +106,6 @@ docker stop docker_import-service_1
 docker-compose down
 ```
 
-You can use a REST client to hit end-points, e.g.
```
Comment (malaffoon) [inline README.md]: This is covered in the API.md file.

File: task-service/src/main/java/org/opentestsystem/rdw/ingest/tasking/service/impl/RestImportServiceClient.java [hunk 2]
```diff
@@ -7,15 +7,12 @@
 import org.opentestsystem.rdw.ingest.tasking.task.UpdateOrganizationsConfiguration;
```
Comment (malaffoon) [inline task-service/src/main/java/org/opentestsystem/rdw/ingest/tasking/service/impl/RestImportServiceClient.java]: just fixing warnings in here

File: task-service/src/main/java/org/opentestsystem/rdw/ingest/tasking/service/impl/ImportServiceClientProperties.java [hunk 3]
```diff
@@ -1,12 +1,14 @@
 package org.opentestsystem.rdw.ingest.tasking.service.impl;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.security.oauth2.client.token.grant.client.ClientCredentialsResourceDetails;
+import org.springframework.security.oauth2.client.token.grant.password.ResourceOwnerPasswordResourceDetails;
```
Comment (malaffoon) [inline task-service/src/main/java/org/opentestsystem/rdw/ingest/tasking/service/impl/ImportServiceClientProperties.java]: this is the main change: the import-service end-point requires user credentials, not just client credentials.

File: README.md [hunk 4]
```diff
@@ -4,11 +4,14 @@ RDW ingest applications:
 1. Exam Processor - Spring Cloud Stream application for processing test results.
 1. Package Processor - Spring Cloud Stream application for processing Assessment Packages.
 1. Task Service - Spring Boot application for running scheduled tasks.
+1. Migrate Reporting - Spring Boot application for migrating data from the warehouse to the reporting data mart.
```
Comment (agorina) [inline README.md]: Thank you for adding it.

Comment (chdietz) [inline README.md]: reminds me we will need documentation on migrate reporting

Comment (malaffoon) [inline README.md]: Yeah, my current philosophy is to pack the README files with lots of information that can be used to write the real documentation when the time comes.

**Pull request merged**